py/torch_tensorrt/dynamo/conversion/impl/elementwise/ops.py

```diff
@@ -213,6 +216,178 @@ def remainder(
     return fmod2_value
 
 
+def atan2(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    other: TRTTensor,
```

Comment:
Does this function support `other` being a Python float or does it require `TRTTensor`?

Reply:
I think it requires `TRTTensor` because both input and output are `Tensor`

![image](https://github.com/pytorch/TensorRT/assets/25674295/007179b0-024e-44fe-93da-6d754d63bf14)